AttributeError: 'dict' object has no attribute 'start'

```
analytics_1  | 2019-04-10 16:55:04,351 [Analytics] [ERROR]  handle_analytic_task Exception: 'Traceback (most recent call last):
analytics_1  |   File "bin/../analytics/analytic_unit_manager.py", line 82, in handle_analytic_task
analytics_1  |     result_payload = await self.__handle_analytic_task(task)
analytics_1  |   File "bin/../analytics/analytic_unit_manager.py", line 69, in __handle_analytic_task
analytics_1  |     return await worker.do_train(payload['segments'], data, payload['cache'])
analytics_1  |   File "bin/../analytics/analytic_unit_worker.py", line 41, in do_train
analytics_1  |     new_cache: ModelCache = await asyncio.wait_for(self._training_future, timeout = config.LEARNING_TIMEOUT)
analytics_1  |   File "/usr/local/lib/python3.6/asyncio/tasks.py", line 358, in wait_for
analytics_1  |     return fut.result()
analytics_1  |   File "/usr/local/lib/python3.6/concurrent/futures/thread.py", line 56, in run
analytics_1  |     result = self.fn(*self.args, **self.kwargs)
analytics_1  |   File "bin/../analytics/detectors/pattern_detector.py", line 49, in train
analytics_1  |     new_cache = self.model.fit(dataframe, segments, self.analytic_unit_id, cache)
analytics_1  |   File "bin/../analytics/models/model.py", line 79, in fit
analytics_1  |     logging.debug(f'segment {segment_map.start}-{segment_map.end} skip because of invalid data')
analytics_1  | AttributeError: 'dict' object has no attribute 'start'
analytics_1  | '
```

--- a/analytics/analytics/models/model.py
+++ b/analytics/analytics/models/model.py
@@ -76,7 +76,7 @@ class Model(ABC):
             if segment_map['labeled'] or segment_map['deleted']:
                 segment = Segment(dataframe, segment_map, self.find_segment_center)
                 if segment.percent_of_nans > 0.1 or len(segment.data) == 0:
-                    logging.debug(f'segment {segment_map.start}-{segment_map.end} skip because of invalid data')
+                    logging.debug(f'segment {segment.start}-{segment.end} skip because of invalid data')
                     continue
                 if segment.percent_of_nans > 0:
                     segment.convert_nan_to_zero()

